deps.go

```diff
@@ -0,0 +1,2 @@
+package deps
+import "github.com/tektoncd/plumbing/scripts"
```

Comment:
`import _ "github.com/tektoncd/plumbing/scripts"` to remove the compilation error.

See https://github.com/tektoncd/pipeline/blob/main/hack/tools.go for examples of "this" in pipeline.